src/components/Purses.jsx

```diff
@@ -65,9 +65,24 @@ export const PursesWithoutContext = ({
     <Loading defaultMessage="Fetching purses..." />
   );
 
+  const helptip = (
+    <span>
+      The smart wallet only supports certain assets and IBC denoms. If you are
+      missing assets it may be because they are not supported.{' '}
+      <a
+        href="https://docs.inter.trade/inter-protocol-system-documentation/parity-stability-module/supported-assets"
```

Comment:
this link is to a PSM resource. the smart wallet is outside PSM and even Inter Protocol. I think this Smart Wallet UI must not prefer any particular dapps or protocols.

Can this information be put somewhere neutral? E.g. on this repo?